src/content/cookbook/architecture/command.md

```diff
@@ -0,0 +1,616 @@
+---
+title: Command pattern
+description: Improve ViewModels with Commands
+js:
+  - defer: true
+    url: /assets/js/inject_dartpad.js
+---
+
+<?code-excerpt path-base="cookbook/architecture/command"?>
+
+Model-View-ViewModel (MVVM) is a design pattern 
+that separates a feature of an application into three parts: 
+the `Model`, the `ViewModel` and the `View`. 
+Views and ViewModels make up the UI layer of an application. 
```

Comment:
I agree, I copied this paragraph from https://docs.flutter.dev/app-architecture/guide#mvvm so that it would be consistent, it capitalizes those words and uses camel case for "ViewModel".

> So, can you (globally) change "Model" to "model", "View" to "view", and "ViewModel" to "view model".

 I will take a look at the whole app-architecture documentation and try to make it consistent, I will do it in this PR.

Reply:
Thanks! Yes, the issue is the same in the architecture docs.

Reply:
Addressed in https://github.com/flutter/website/pull/11458